src/components/mapSettingsMenu/MenuActivator.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+    <v-theme-provider theme="dark">
+        <v-btn
+            v-if="appConfig && mapSettings.indicator"
+            role="button"
+            :class="props.isLargeScreen ? 'indicator-menu-activator-desktop' : 'indicator-menu-activator-mobile'"
+        >
+            <p class="text-wrap">
+                {{ buttonSummary }}
+            </p>
+            <v-icon :icon="menuOpen ? 'mdi-chevron-down' : 'mdi-chevron-up'" end></v-icon>
```

Comment:
I know you probably introduce this, but to me these down and up seem back to front to what I'd expect.

Reply:
I'd expect `“up” state when expanded and the “down” state when collapsed` as suggested [here](https://primer.style/ui-patterns/progressive-disclosure#chevron-icon)

Reply:
That's a drop-down menu though, rather than an expand-up. My intuition was that the chevron should point in the direction that the items will expand/contract into. Google maps does something similar: 
[Screencast from 06-06-24 10:35:47.webm](https://github.com/mrc-ide/arbomap/assets/44669576/bb2c611c-fdef-4ff4-a962-e3fadb13ad22)

But maybe 'down for expand' is also expected whichever the direction?

Reply:
I don't feel strongly about it, and it's fundamentally about intuitions isn't it

Reply:
i dont feel too strongly about this either, so will leave this as is for now, i think its more important that the arrows toggle than which particular states